api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/BaseQueries.java

```diff
@@ -24,8 +24,8 @@
 	// they map to concept_id=1369 - TRANSFER FROM OTHER FACILITY
 	// TODO: Query needs to be refactored
 	public static String getBaseCohortQuery(Map<String, String> parameters) {
-		String query = "select p.patient_id from patient p join encounter e on e.patient_id=p.patient_id join person pr on pr.person_id = p.patient_id "
```

Comment:
We didn't add this birthdate check here, it was already in the base query. Should we be removing it?

Reply:
Sorry @nthfloor I'm not sure I understood. We added the `null birthdate` check to base cohort to complete version 1 of the reports and them removed to make the Unknown disaggregation possible.

Reply:
The null check was only added at the end on L38 [here](https://github.com/esaude/openmrs-module-eptsreports/blob/36e49169bf5a4732df094f7d3cd6eb5a92c06517/api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/BaseQueries.java#L37-L38). You can see how the base query was joined to person table and the birthdate check was added.

But the birthdate check on L29 was always part of the original query.

Reply:
Oh, now I understand. Actually I added that line too by mistake. It was my first attempt to implement the exclusion but was not correct. I added it here https://github.com/esaude/openmrs-module-eptsreports/commit/4848f64fd90173d70fbc710b6f254722bd16df9b#diff-4e3e6ce8ab9071a025d7c13568148a14

Reply:
Ah I see, ok cool :+1: